Sources/Security.php

```diff
@@ -465,25 +465,22 @@ function banPermissions()
 	}
 
 	// Now that we have the mod cache taken care of lets setup a cache for the number of mod reports still open
-	if (!empty($_SESSION['rc']) && $_SESSION['rc']['time'] > $modSettings['last_mod_report_action'] && $_SESSION['rc']['id'] == $user_info['id'])
+	if (!empty($_SESSION['rc']['reports']) && !empty($_SESSION['rc']['member_reports']) && $_SESSION['rc']['time'] > $modSettings['last_mod_report_action'] && $_SESSION['rc']['id'] == $user_info['id'])
+	{
```

Comment:
Correct me if I'm wrong, but this means the cache is only valid if ['rc']['reports'] and ['rc']['member_reports'] are both not 0, instead of ['rc'] just existing.  So if you have no member reports open, it is going to ignore the cache completely?